scripts/generators/templates/field_details/row.j2

```diff
@@ -0,0 +1,14 @@
+
+| {{ flat_name }}
+| {{ description|replace("\n", "\n\n") }}
```

Comment:
This was the only transformation we had in `render_asciidoc_paragraphs()` so that's fine for now.

But if we decide to introduce more transformations again (e.g. support hyperlinks), do you know how to create these Jinja2 modifier functions?

Reply:
Yes. Outside of the [built-in filters](https://jinja.palletsprojects.com/en/2.11.x/templates/#list-of-builtin-filters), we can also define [custom filters](https://jinja.palletsprojects.com/en/2.11.x/api/#custom-filters) by passing our own defined functions into the Jinja environment declaration.